packs/gcp_k8.yml

```diff
@@ -0,0 +1,21 @@
+AnalysisType: pack
+PackID: PantherManaged.GCP.K8
+DisplayName: "Panther GCP Kubernetes Pack"
+Description: Group of all Google Cloud Platform (GCP) K8 detections
+PackDefinition:
+  IDs:
+    # DataModel
+    - Standard.GCP.AuditLog
+    # Rules
```

Comment:
Let's move https://github.com/panther-labs/panther-analysis/blob/release/packs/gcp_audit.yml#L30 here

Reply:
Are you saying move gcp_k8 -> gcp_audit, or gcp_audit -> gcp_k8? Which is the pack we'll keep?

Reply:
Oh sorry, I meant move the 1 gcp_k8s rule from the gcp_audit pack to the gcp_k8s pack.